src/Forms/FieldList.php

```diff
@@ -410,12 +385,12 @@ public function findTab($tabName)
      * @param string $tabName The tab to return, in the form "Tab.Subtab.Subsubtab".
      *   Caution: Does not recursively create TabSet instances, you need to make sure everything
      *   up until the last tab in the chain exists.
-     * @param string $title Natural language title of the tab. If {@link $tabName} is passed in dot notation,
+     * @param string|null $title Natural language title of the tab. If {@link $tabName} is passed in dot notation,
      *   the title parameter will only apply to the innermost referenced tab.
      *   The title is only changed if the tab doesn't exist already.
      * @return Tab The found or newly created Tab instance
      */
-    public function findOrMakeTab($tabName, $title = null)
+    public function findOrMakeTab(string $tabName, ?string $title = null): mixed
```

Comment:
Mixed doesn't seem right - and doesn't match the phpdoc. 
```suggestion
    public function findOrMakeTab(string $tabName, ?string $title = null): Tab
```
This method is for finding or making a tab. If the object we're returning isn't a tab, we need to update the logic.

Reply:
Done